### PR TITLE
Incorrect information regarding VolumeSnapshot API corrected

### PR DESCRIPTION
**Pull request by NitishKumar06**
Fix : #34143

### CODE REVIEW

File: content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ been deprecated. These removals have been superseded by newer, stable/generally
...
+* [VolumeSnapshot v1beta1 CRD has been removed](https://github.com/kubernetes/enhancements/issues/177). Volume snapshot and restore functionality for Kubernetes and the [Container Storage Interface](https://github.com/container-storage-interface/spec/blob/master/spec.md) (CSI), which provides standardized APIs design (CRDs) and adds PV snapshot/restore support for CSI volume drivers, moved to GA in v1.20. VolumeSnapshot v1beta1 was deprecated in v1.20 and is now unsupported. Refer to [KEP-177: CSI Snapshot](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/177-volume-snapshot#kep-177-csi-snapshot) and [Volume Snapshot GA blog](https://kubernetes.io/blog/2020/12/10/kubernetes-1.20-volume-snapshot-moves-to-ga/) for more information.
```
[the remark this answers is not in the document]
Comment (reylejano) [reply content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md:76]: Hi @NitishKumar06 , @Sea-n 's suggestion is correct
We use reference paths for kubernetes.io

[the remark this answers is not in the document]
Comment (NitishKumar06) [reply content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md:76]: Okay I'll add the same

**Comment by NitishKumar06** [comment]
Hey @reylejano @xing-yang I'm actually going to close this PR since I've made some wrong commits. I'll raise a new PR adding the changes you suggested! Thank you!

**Comment by NitishKumar06** [comment]
Please refer to this https://github.com/kubernetes/website/pull/34173

File: content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md [hunk 2]
```diff
@@ -39,71 +39,71 @@ to container runtimes that are directly compatible with Kubernetes. You can find
...
+* [VolumeSnapshot v1beta1 CRD has been removed](https://github.com/kubernetes/enhancements/issues/177). Volume snapshot and restore functionality for Kubernetes and the [Container Storage Interface](https://github.com/container-storage-interface/spec/blob/master/spec.md) (CSI), which provides standardized APIs design (CRDs) and adds PV snapshot/restore support for CSI volume drivers, moved to GA in v1.20. VolumeSnapshot v1beta1 was deprecated in v1.20 and is now unsupported. Refer to [KEP-177: CSI Snapshot](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/177-volume-snapshot#kep-177-csi-snapshot) and [Volume Snapshot GA blog](/blog/2020/12/10/kubernetes-1.20-volume-snapshot-moves-to-ga/) for more information.
```
Comment (xing-yang) [inline content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md:76]: This change looks good now.

**Pull request closed without merging**